src/pages/additional-features/blueprint.mdx

```diff
@@ -0,0 +1,34 @@
+import { Callout } from 'nextra-theme-docs'
+
+# Blueprint
+
+Plutus Contract Blueprint, as introduced by [CIP-57](https://cips.cardano.org/cip/CIP-0057) is an effective way of communicating desired structure of validator's associated types such as datum & redeemer to your off-chain code.
```

Comment:
The phrase "desired structure of validator's associated types such as datum & redeemer to your off-chain code" feels awkward. It could be rephrased to improve flow:

"...communicating the desired structure of a validator’s associated types, such as datum and redeemer, to your off-chain code."